doc/boundary_conditions.rst

```diff
@@ -0,0 +1,131 @@
+.. _Boundary conditions:
+
+Boundary Conditions
+-------------------
+
+.. ipython:: python
+
+    import xarray as xr
+    import numpy as np
+    from xgcm import Grid
+
+    ds = xr.Dataset(
+        coords={
+            "x_c": (
+                [
+                    "x_c",
+                ],
+                np.arange(1, 10),
+                {"axis": "X"},
+            ),
+            "x_g": (
+                [
+                    "x_g",
```

Comment:
See my comment in [#464](https://github.com/xgcm/xgcm/pull/464#discussion_r852342050). I think this can be made more compact.

Reply:
Let's just deal with all docs formatting in one go in another PR.